app/i18n/en/namespaces/pages.js

```diff
@@ -10,6 +10,9 @@ const pages = {
   topic_new: {
     title: 'Create new topic',
   },
+  profile: {
```

Comment:
General remark: we do need to figure out a consistent schema for the localization files. See #20.